.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: npm run test:unit --prefix=app/static
       - name: Run browser tests
         run: |
-          npx playwright install
+          (cd app/static && npx playwright install)
```

Comment:
This change fixes a break in the build which is also affecting `main` branch - it seems the latest version of Playwright requires to be installed in the same folder as the package being tested.